features/product/managing_product_variants/sorting_product_variants_within_product_by_position.feature

```diff
@@ -0,0 +1,55 @@
+@managing_product_variants
+Feature: Sorting listed product variants from a product by position
+    In order to change the order by which product variants from a product are displayed
+    As an Administrator
+    I want to sort product variants from a product by their positions
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has an "Opel Insignia" configurable product
+        And this product has an "Opel Insignia Hatchback" variant at position 0
+        And this product has also an "Opel Insignia Sedan" variant at position 1
+        And this product has also an "Opel Insignia Sports Tourer" variant at position 2
```

Comment:
We should shuffle this variants to be sure they aren't simply sorted by their ids.